Traceback (most recent call last):
  File "/Users/johannes/opt/anaconda3/envs/books/lib/python3.8/site-packages/jupyter_cache/executors/utils.py", line 51, in single_nb_execution
    executenb(
  File "/Users/johannes/opt/anaconda3/envs/books/lib/python3.8/site-packages/nbclient/client.py", line 1087, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/Users/johannes/opt/anaconda3/envs/books/lib/python3.8/site-packages/nbclient/util.py", line 74, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/Users/johannes/opt/anaconda3/envs/books/lib/python3.8/site-packages/nbclient/util.py", line 53, in just_run
    return loop.run_until_complete(coro)
  File "/Users/johannes/opt/anaconda3/envs/books/lib/python3.8/asyncio/base_events.py", line 608, in run_until_complete
    return future.result()
  File "/Users/johannes/opt/anaconda3/envs/books/lib/python3.8/site-packages/nbclient/client.py", line 540, in async_execute
    await self.async_execute_cell(
  File "/Users/johannes/opt/anaconda3/envs/books/lib/python3.8/site-packages/nbclient/client.py", line 832, in async_execute_cell
    self._check_raise_for_error(cell, exec_reply)
  File "/Users/johannes/opt/anaconda3/envs/books/lib/python3.8/site-packages/nbclient/client.py", line 740, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply['content'])
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
from gensim.models import KeyedVectors

# Creating the model
#en_model = KeyedVectors.load_word2vec_format('/Users/maucher/DataSets/Gensim/FastText/Gensim/FastText/wiki-news-300d-1M.vec')
#en_model = KeyedVectors.load_word2vec_format(r'C:\Users\maucher\DataSets\Gensim\Data\Fasttext\wiki-news-300d-1M.vec\wiki-news-300d-1M.vec') #path on surface
en_model = KeyedVectors.load_word2vec_format('/Users/johannes/DataSets/Gensim/FastText/fasttextEnglish300.vec')
# Getting the tokens 
words = []
for word in en_model.key_to_index:
    words.append(word)

# Printing out number of tokens available
print("Number of Tokens: {}".format(len(words)))

# Printing out the dimension of a word vector 
print("Dimension of a word vector: {}".format(
    len(en_model[words[0]])
))
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mValueError[0m                                Traceback (most recent call last)
[0;32m<ipython-input-10-f7ca27df6c4c>[0m in [0;36m<module>[0;34m[0m
[0;32m----> 1[0;31m [0;32mfrom[0m [0mgensim[0m[0;34m.[0m[0mmodels[0m [0;32mimport[0m [0mKeyedVectors[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m      2[0m [0;34m[0m[0m
[1;32m      3[0m [0;31m# Creating the model[0m[0;34m[0m[0;34m[0m[0;34m[0m[0m
[1;32m      4[0m [0;31m#en_model = KeyedVectors.load_word2vec_format('/Users/maucher/DataSets/Gensim/FastText/Gensim/FastText/wiki-news-300d-1M.vec')[0m[0;34m[0m[0;34m[0m[0;34m[0m[0m
[1;32m      5[0m [0;31m#en_model = KeyedVectors.load_word2vec_format(r'C:\Users\maucher\DataSets\Gensim\Data\Fasttext\wiki-news-300d-1M.vec\wiki-news-300d-1M.vec') #path on surface[0m[0;34m[0m[0;34m[0m[0;34m[0m[0m

[0;32m~/opt/anaconda3/envs/books/lib/python3.8/site-packages/gensim/__init__.py[0m in [0;36m<module>[0;34m[0m
[1;32m      9[0m [0;32mimport[0m [0mlogging[0m[0;34m[0m[0;34m[0m[0m
[1;32m     10[0m [0;34m[0m[0m
[0;32m---> 11[0;31m [0;32mfrom[0m [0mgensim[0m [0;32mimport[0m [0mparsing[0m[0;34m,[0m [0mcorpora[0m[0;34m,[0m [0mmatutils[0m[0;34m,[0m [0minterfaces[0m[0;34m,[0m [0mmodels[0m[0;34m,[0m [0msimilarities[0m[0;34m,[0m [0mutils[0m  [0;31m# noqa:F401[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m     12[0m [0;34m[0m[0m
[1;32m     13[0m [0;34m[0m[0m

[0;32m~/opt/anaconda3/envs/books/lib/python3.8/site-packages/gensim/corpora/__init__.py[0m in [0;36m<module>[0;34m[0m
[1;32m      4[0m [0;34m[0m[0m
[1;32m      5[0m [0;31m# bring corpus classes directly into package namespace, to save some typing[0m[0;34m[0m[0;34m[0m[0;34m[0m[0m
[0;32m----> 6[0;31m [0;32mfrom[0m [0;34m.[0m[0mindexedcorpus[0m [0;32mimport[0m [0mIndexedCorpus[0m  [0;31m# noqa:F401 must appear before the other classes[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m      7[0m [0;34m[0m[0m
[1;32m      8[0m [0;32mfrom[0m [0;34m.[0m[0mmmcorpus[0m [0;32mimport[0m [0mMmCorpus[0m  [0;31m# noqa:F401[0m[0;34m[0m[0;34m[0m[0m

[0;32m~/opt/anaconda3/envs/books/lib/python3.8/site-packages/gensim/corpora/indexedcorpus.py[0m in [0;36m<module>[0;34m[0m
[1;32m     12[0m [0;32mimport[0m [0mnumpy[0m[0;34m[0m[0;34m[0m[0m
[1;32m     13[0m [0;34m[0m[0m
[0;32m---> 14[0;31m [0;32mfrom[0m [0mgensim[0m [0;32mimport[0m [0minterfaces[0m[0;34m,[0m [0mutils[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m     15[0m [0;34m[0m[0m
[1;32m     16[0m [0mlogger[0m [0;34m=[0m [0mlogging[0m[0;34m.[0m[0mgetLogger[0m[0;34m([0m[0m__name__[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m

[0;32m~/opt/anaconda3/envs/books/lib/python3.8/site-packages/gensim/interfaces.py[0m in [0;36m<module>[0;34m[0m
[1;32m     17[0m [0;32mimport[0m [0mlogging[0m[0;34m[0m[0;34m[0m[0m
[1;32m     18[0m [0;34m[0m[0m
[0;32m---> 19[0;31m [0;32mfrom[0m [0mgensim[0m [0;32mimport[0m [0mutils[0m[0;34m,[0m [0mmatutils[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m     20[0m [0;34m[0m[0m
[1;32m     21[0m [0;34m[0m[0m

[0;32m~/opt/anaconda3/envs/books/lib/python3.8/site-packages/gensim/matutils.py[0m in [0;36m<module>[0;34m[0m
[1;32m   1022[0m [0;32mtry[0m[0;34m:[0m[0;34m[0m[0;34m[0m[0m
[1;32m   1023[0m     [0;31m# try to load fast, cythonized code if possible[0m[0;34m[0m[0;34m[0m[0;34m[0m[0m
[0;32m-> 1024[0;31m     [0;32mfrom[0m [0mgensim[0m[0;34m.[0m[0m_matutils[0m [0;32mimport[0m [0mlogsumexp[0m[0;34m,[0m [0mmean_absolute_difference[0m[0;34m,[0m [0mdirichlet_expectation[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m   1025[0m [0;34m[0m[0m
[1;32m   1026[0m [0;32mexcept[0m [0mImportError[0m[0;34m:[0m[0;34m[0m[0;34m[0m[0m

[0;32m~/opt/anaconda3/envs/books/lib/python3.8/site-packages/gensim/_matutils.pyx[0m in [0;36minit gensim._matutils[0;34m()[0m

[0;31mValueError[0m: numpy.ndarray size changed, may indicate binary incompatibility. Expected 88 from C header, got 80 from PyObject
ValueError: numpy.ndarray size changed, may indicate binary incompatibility. Expected 88 from C header, got 80 from PyObject

